README.rst

```diff
@@ -17,10 +17,7 @@ Introduction
 .. image:: https://raw.githubusercontent.com/kiwitcms/Kiwi/master/tcms/static/images/kiwi_h80.png
    :alt: "Kiwi TCMS Logo"
 
-Kiwi TCMS is a test plan, test run and test case management system, that is
-written in Python and Django. It has a lot of great features, such as Bugzilla
-and JIRA integration, fast test plan and runs search, powerful access control
-for each plan, run and case, and XML-RPC APIs.
+Kiwi TCMS is a test plan, test run and test case management system, written in Python and Django. It features Bugzilla and JIRA integration, fast test plan and runs search, powerful access control for each plan, run and case, and XML-RPC APIs.
```

Comment:
Wording looks good but please try to wrap the text around the 80th column and use newlines.

Reply:
🆗